lib/index.js

```diff
@@ -72,6 +72,8 @@ module.exports = class ArkPay extends Emittery {
      */
     vendorField (value) {
       this.data.transfer.vendorField = value
+      
+      if (value.length > 64) this.emit('error', 'Value in venderfield is too long')
```

Comment:
The emitter is not yet used at this point. It should simply throw an error.

Reply:
Also the wording is odd, should be `The given value exceeds the maximum length of 64 characters.`.